lib/git_worktree.rb

```diff
@@ -37,17 +37,11 @@ def initialize(options = {})
     @remote_name = 'origin'
     @base_name   = File.basename(@path)
 
-    # The libssh2 library must already be installed at gem installation time
-    # for the 'ssh' feature to be available.
+    # The libssh2 library must already be installed and the `--with-ssh` build option
+    # passed at gem installation time for the 'ssh' feature to be available.
     #
-    # For Fedora/Centos, the presence of libssh2 seems to be enough to get it
-    # to build properly.
-    #
-    # For OSX:
-    #
-    #     $ brew install libssh2
```

Comment:
I left this out since we are going to add it to the developer_setup guide, plus it wasn't present for Fedora/CentOS and adding `dnf install libssh2-devel` didn't seem necessary